third_party/v8/src/base/platform/time.cc

```diff
@@ -765,9 +765,7 @@ bool ThreadTicks::IsSupported() {
 
 ThreadTicks ThreadTicks::Now() {
 #if V8_OS_STARBOARD
-  if (SbTimeIsTimeThreadNowSupported())
-    return ThreadTicks(SbTimeGetMonotonicThreadNow());
-  UNREACHABLE();
```

Comment:
Why do we want to get rid of the UNREACHABLE block if this is being called while CurrentMonotonicThreadTime is unsupported?

Reply:
Would this affect performance at all given that the UNREACHABLE block should never actually be executed here (and I assume does nothing in gold builds anyways)? It seems to me to just be a safety net for non-prod builds so that clients don't incorrectly use it.